scripts/main.js

```diff
@@ -45,10 +47,31 @@ function studentContact(record) {
 }
 
 
```

Comment:
Updated this to be a link instead of a button, and made it present always. If there isn't a resume available, it's disabled instead of invisible